docs/root/configuration/http/http_filters/ip_tagging_filter.rst

```diff
@@ -3,9 +3,9 @@
 IP Tagging
 ==========
 
-The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* with the string tags for the trusted address from
-:ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`. If there are no tags for an address,
-the header is not set.
+The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* if the alternate :ref: `ip_tag_header <envoy_v3_api_msg_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
```

Comment:
```suggestion
The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* if the alternate :ref: `ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
```

otoh, try this to see if it helps https://github.com/envoyproxy/envoy/pull/36434#discussion_r1804991509 @Radha13 ?

Reply:
Radha is unavailable at the moment for personal reasons, so I'm taking over. <3
Although I don't have write access to the repository from which the pull request originates, so it might be a few days before we can sort that. <3